### PR TITLE
Feature:Dockerize code Upload worker

### PR DESCRIPTION
**Pull request by Ayukha**
**Why do we need it?**
We want to create a docker image of the code-upload-worker to run it on AWS Fargate

**Changes Proposed:**
Dockerizes the Code upload worker and pushes the docker image to ECR

### CODE REVIEW

File: docker/dev/code-upload-worker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+RUN mkdir /code
+WORKDIR /code
+
+ADD requirements/* /code/
+
+RUN pip install -r rl-worker.txt
+
+ADD . /code
+
+CMD ["./docker/wait-for-it.sh", "django:8000", "--", "python", "-m", "scripts.workers.rl_submission_worker"]
```
Comment (RishabhJain2018) [inline docker/dev/code-upload-worker/Dockerfile:14]: Add a new line here.

File: docker/dev/docker_code_upload_worker.env [hunk 2]
```diff
@@ -0,0 +1,4 @@
+AUTH_TOKEN=x
+DJANGO_SERVER=django
+DJANGO_SERVER_PORT=8000
+QUEUE_NAME=evalai_submission_queue
```
Comment (RishabhJain2018) [inline docker/dev/docker_code_upload_worker.env:4]: Add a new line here.

File: docker/prod/code-upload-worker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+RUN mkdir /code
+WORKDIR /code
+
+ADD requirements/* /code/
+
+RUN pip install -r rl-worker.txt
+
+ADD . /code
+
+CMD ["./docker/wait-for-it.sh", "django:8000", "--", "python", "-m", "scripts.workers.rl-submission_worker"]
```
Comment (RishabhJain2018) [inline docker/prod/code-upload-worker/Dockerfile:14]: Add a new line here?

File: docker/prod/docker_code_upload_worker.env [hunk 4]
```diff
@@ -0,0 +1,4 @@
+AUTH_TOKEN=x
+DJANGO_SERVER=django
+DJANGO_SERVER_PORT=8000
+QUEUE_NAME=evalai_submission_queue
```
Comment (RishabhJain2018) [inline docker/prod/docker_code_upload_worker.env:4]: Add a new line here.

File: requirements/rl-worker.txt [hunk 5]
```diff
@@ -0,0 +1,2 @@
+kubernetes==10.0.1
```
Comment (RishabhJain2018) [inline requirements/rl-worker.txt:1]: change the filename to `code_upload_worker.txt`

File: docker/dev/docker_code_upload_worker.env (hunk 2 above)
Comment (RishabhJain2018) [reply docker/dev/docker_code_upload_worker.env:4]: @Ayukha Can you please quickly make this change.

**Comment by RishabhJain2018** [comment]
@Ayukha Can you please PR description.

**Comment by Ayukha** [comment]
@RishabhJain2018  Done

**Pull request merged**